dev-docs/bidders/connatix.md

```diff
@@ -57,3 +57,24 @@ var adUnits = [
   // Define more ad units here if necessary
 ];
 ```
+
+### Configuration
+
+To maximize revenue efficiency, please enable ```iframe``` user syncing.
+
+Connatix strongly recommends enabling user syncing through iFrames. This functionality improves DSP user match rates and increases the bid rate and bid price. Make sure to call `pbjs.setConfig()` only once. This configuration is optional in Prebid, but required by Connatix.
+
+#### Example configuration
+
+```js
+pbjs.setConfig({
+  userSync: {
+    filterSettings: {
+      iframe: {
+        bidders: '*', // represents all bidders

```

Comment:
This will enable iframe syncs for all bidders. I would prefer a more subtle example like this

```suggestion
        bidders: ['connactix'],

```